src/main/java/edu/pw/eiti/pik/participation/ParticipationChangeStatus.java

```diff
@@ -0,0 +1,11 @@
+package edu.pw.eiti.pik.participation;
+
+public enum ParticipationChangeStatus {
```

Comment:
I think you can use already existing ParticipationStatus enum. In path variable set the participation status you would like to set. Maybe extension of existing enum is required (p. ex. add "resigned" status, use it as path variable and then delete participation)